src/t_string.c

```diff
@@ -595,7 +595,7 @@ void msetnxCommand(client *c) {
     msetGenericCommand(c,1);
 }
 
-void incrDecrCommand(client *c, long long incr) {
+void incrDecrCommand(client *c, long long incr, char * const event) {
```

Comment:
Constant value instead of pointer?
```suggestion
void incrDecrCommand(client *c, long long incr, const char *event) {
```